tfhe/src/core_crypto/gpu/vec.rs

```diff
@@ -175,16 +175,13 @@ impl<T: Numeric> CudaVec<T> {
     ///
     /// - `streams` __must__ be synchronized to guarantee computation has finished, and inputs must
     ///   not be dropped until streams is synchronised
-    pub unsafe fn memset_async(&mut self, value: T, streams: &CudaStreams, stream_index: u32)
-    where
-        T: Into<u64>,
-    {
+    pub unsafe fn memset_async(&mut self, value: u64, streams: &CudaStreams, stream_index: u32) {
```

Comment:
no adverse effect for your usage ?

Reply:
No actually we were always calling it with u64 :sweat_smile: